quick-protobuf/examples/pb_rs/data_types.rs

```diff
@@ -49,11 +49,20 @@ impl<'a> From<&'a str> for FooEnum {
 }
 
 #[allow(clippy::derive_partial_eq_without_eq)]
-#[derive(Debug, Default, PartialEq, Clone)]
+#[derive(Debug, PartialEq, Clone)]
 pub struct BarMessage {
     pub b_required_int32: i32,
 }
 
+
+impl Default for BarMessage {
+    fn default() -> Self {
+        Self {
+            b_required_int32: 0i32,
+        }
+    }
+}
```

Comment:
Is this change strictly necessary? I'd be nice if the generated code is as idiomatic as possible.

One reason we are moving to this library from protoc is because we will commit the generated code to Git. Makes the build process simpler, IDE integration is better and the code is actually decent to look at.

Reply:
Made a change that will avoid custom `impl Default` on structs that don't need it; should be present in next update